packages/manager-components/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@ovhcloud/manager-components",
+  "private": true,
+  "version": "1.0.0",
+  "main": "src/components/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager-components"
+  },
+  "scripts": {
+    "start": "storybook dev -p 6006",
+    "dev": "tsc",
+    "build": "echo manager-components",
+    "prettier": "prettier --write \"src/**/*.{ts,tsx,js,mdx}\"",
+    "test": "jest --coverage -u",
+    "test:cov": "vitest run --coverage",
+    "test:e2e": "tsc && npx playwright test --ui --debug"
+  },
+  "dependencies": {
+    "babel-preset-react-app": "^10.0.1",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.22.10",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@ovh-ux/manager-tailwind-config": "*",
+    "@ovhcloud/ods-common-core": "^16.6.0",
+    "@ovhcloud/ods-theme-blue-jeans": "^16.6.0",
+    "@ovhcloud/ods-common-stencil": "^16.6.0",
+    "@ovhcloud/ods-common-testing": "^16.6.0",
+    "@ovhcloud/ods-common-theming": "^16.6.0",
+    "@ovhcloud/ods-components": "^16.6.0",
```

Comment:
**question:** Why ODS is made as a devdependency? It must be dependency right?

Reply:
yes I move it

Reply:
👍